tests/suites/test_suite_x509parse.function

```diff
@@ -762,7 +762,7 @@ void x509_parse_rsassa_pss_params( data_t *hex_params, int params_tag,
     }
 
 exit:
-    ;;
+    ;
```

Comment:
The empty `exit` label can be removed completely, the test framework will add it automatically, but this doesn't hurt anything.